README.md

```diff
@@ -1,23 +1,36 @@
 # boltzmann
 
-A catchy slogan here.
+Boltzmann is a JS micro framework for JSON api services. It is implemented in a single file that lives alongside your code.
 
-Boltzmann doesn't try to be fast. We have no idea how fast or how slow Boltzmann is, in fact. (If we needed to be fast, we'd use Rust.) We are focused on *developer experience*. Things that we find we need to do every time we write a web service, we want our framework to do out of the box. We also do not want to invent more than we need to. Boltzmann is a thin wrapper around well-tested http primitives, such as [jshttp project](https://github.com/brianc/node-postgres) and [find-my-way](https://github.com/brianc/node-postgres). Our goal is that it should be easy to _throw Boltzmann away_ if you move on from it.
+Boltzmann doesn't try to be fast. We have no idea how fast or how slow Boltzmann is, in fact. (If we needed to be fast, we'd use Rust.) We are focused on *developer experience*. Things that we find we need to do every time we write a service, we want our framework to do out of the box. We also do not want to invent more than we need to, so Boltzmann is a thin wrapper around well-tested node packages like [jshttp](https://github.com/jshttp) and [find-my-way](https://github.com/delvedor/find-my-way).
```

Comment:
We're starting out a bit defensive here. I suggest bringing "Boltzmann is focused on delivering a great *developer experience*." up into the previous para, and dropping the rest in favor of the design goals below. We may add "Use only minimal, well-vetted dependencies" to the design goals as well.